docs/connector-development/connector-builder-ui/maintenance.md

```diff
@@ -0,0 +1,175 @@
+* Create tests scenarios in a sandbox environment: There is no streamlined process to do this. Each product has their own business rules that can't be captured in a unifying document. Given a new connector, this process might require to have pay for a subscription for the product in order to get credentials/workspace/etc... In order to be able to repopulate an API very easily, we suggest the use of a Postman collection or a script.
+* Configure what we want to be validated: Using `acceptance-test-config.yml`, you can toggle some of the validations that are performed. The most common cases are:
+    * If `expected_records.jsonl` to validate the records that the source returns has been added, ensure `acceptance_tests.basic_read.tests.expect_records` is populated. This might require you to create a catalog for this test suites.
+    * If a stream was made incremental, ensure `acceptance_tests.incremental.tests.expect_records` is populated. This might require you to create a catalog for this test suites.
+* Set up the configuration: In order to run properly, CATs require a valid config. Those configs often rely on secret. You can use [GSM](https://console.cloud.google.com/security/secret-manager?project=dataline-integration-testing) to modify the existing configuration. Simply write the connector name in the filter bar, click on `SECRET_SOURCE-<name>__CREDS` and `NEW VERSION` and upload the new config.
+
+To execute CATs, there are two options.
+
+### Locally
+We recommend using `cd airbyte-integrations/connections/source-<name> && ./acceptance-test-docker.sh` to run them using docker. You might have to change the permissions to run this file. Once it's done, the results will be printed to the terminal.
+
+### In a GitHub pull request
+Given that you have pushed you changes on a branch in airbytehq/airbyte, go on https://github.com/airbytehq/airbyte and you should see a yellow banner saying "<branch> had recent pushes less than a minute ago". Click on "Compare & pull request", fill out the template and click on "Create pull request". A check named "Connectors tests / Connectors CI" will run. Once it's done, a new comment will be added to show the test results. 
+
+## Release
+
+Goal: Make the connector available to end-users
+
+Expected output: https://hub.docker.com/r/airbyte/source-<name>/tags shows a recently created tag
+
+The following steps assume that you are getting
+
+* Bump version (Docker file and metadata.yaml) and update changelogs under `docs/integrations`
+* Fill the Connector Checklist in the pull request
+* If there are breaking changes, follow the process defined by [Connector Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit#heading=h.r2xzw0kclodc). A breaking change is a change that requires the user to take action. It can be triggered by a change in the schema, in the config or in the connector behavior. 
```

Comment:
I'd expect any connector updates to change the connector behavior. We probably only care about the 5 types of changes listed in the playbook

Reply:
Yes! Reading this, I feel like I should just refer to the document which explains what changes are breaking so that if the list is updated, we don't need to update the `maintenance.md`